src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TypeChar.cs

```diff
@@ -313,7 +313,7 @@ private bool IsTextualTriggerCharacter(ICompletionService completionService, cha
             return completionService.IsTriggerCharacter(previousPosition.Snapshot.AsText(), previousPosition, GetCompletionProviders(), options);
         }
 
-        private bool IsCommitCharacter(char ch)
+        private bool IsCommitCharacter(char typedChar)
```

Comment:
Changing to soft selection sounds reasonable. User in that have a visual to predict what will the IDE do next.